apps/desktop-wallet/src/index.css

```diff
@@ -52,9 +52,6 @@ a {
   color: inherit;
 }
 
-#app-header {
-  -webkit-app-region: drag;
-}
```

Comment:
This is required so the user can drag the window around the screen by clicking on the `AppHeader` component.

If a parent has `-webkit-app-region: drag;`, then a child can have `-webkit-app-region: no-drag;` to allow button clicks.

I suggest that:
1. For Windows, we remove the above 3 lines. For all other systems, we keep them.
2. For macOS and Linux, we add `-webkit-app-region: no-drag;` to `HeaderButtons`